app/templates/main/browse.html

```diff
@@ -193,168 +195,155 @@ <h1 class="govuk-heading-l browse__records-found__text">Records found {{ num_rec
-                                        </dd>
-                                        <dd class="govuk-summary-list__value">
-                                            <p class="govuk-body browse__body">
+                            <table class="govuk-table browse-grid__table">
+                                <thead class="govuk-table__head">
+                                    <tr class="govuk-table__row">
+                                        <th scope="col" class="govuk-table__header">Last modified</th>
+                                        <th scope="col" class="govuk-table__header">Filename</th>
+                                        <th scope="col" class="govuk-table__header">Status</th>
+                                        <th scope="col" class="govuk-table__header">Closure start date</th>
+                                        <th scope="col" class="govuk-table__header">Closure period</th>
+                                    </tr>
+                                </thead>
+                                <tbody class="govuk-table__body">
+                                    {% for record in results %}
+                                        <tr class="govuk-table__row">
+                                            <td class="govuk-table__cell">{{ record["date_last_modified"] }}</td>
+                                            <td class="govuk-table__cell">
+                                                <a href="{{ url_for('main.record', record_id=record['file_id']) }}">{{ record["file_name"] }}</a>
+                                            </td>
+                                            <td class="govuk-table__cell">
+                                                <strong class="govuk-tag {{ 'govuk-tag--green' if record['closure_type'] == 'Open' else 'govuk-tag--red' }}">
+                                                    {{ record['closure_type'] }}
+                                                </strong>
+                                            </td>
+                                            <td class="govuk-table__cell">
+                                                {% if record['closure_start_date'] %}
+                                                    {{ record['closure_start_date'] }}
+                                                {% else %}
+                                                    -
+                                                {% endif %}
+                                            </td>
+                                            <td class="govuk-table__cell">
                                                 {% if record['closure_period'] %}
                                                     {{ record['closure_period'] }} years
                                                 {% else %}
-                                                    None
+                                                    -
```

Comment:
It should be an "-" instead of None / not available. We've got an updated ticket for the actual change of this work to be completed in & the closure period is due to be calculated see:

https://national-archives.atlassian.net/browse/AYR-608
https://national-archives.atlassian.net/browse/AYR-654

Reply:
nice one, cheers